spring-boot-starter-camunda/src/main/java/io/camunda/zeebe/spring/client/properties/CamundaClientProperties.java

```diff
@@ -0,0 +1,110 @@
+package io.camunda.zeebe.spring.client.properties;
+
+import io.camunda.zeebe.spring.client.properties.common.ApiProperties;
+import io.camunda.zeebe.spring.client.properties.common.AuthProperties;
+import io.camunda.zeebe.spring.client.properties.common.ZeebeClientProperties;
+import java.util.List;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+@ConfigurationProperties("camunda.client")
+public class CamundaClientProperties {
+
+  private ClientMode mode;
+  private String clusterId;
+  private String region;
```

Comment:
Is it possible to connect to SaaS environments other than prod with this config? I think it would require overriding the `authUrl`.

Reply:
It is by using the camunda.client.auth.issuer